apps/nextjs/src/components/Icon/svgs.tsx

```diff
@@ -73,77 +73,86 @@ import upload from "@/assets/svg/upload.svg";
 
 import type { IconName } from "./types";
 
-export const svgs: Record<IconName, string> = {
-  "arrow-left": arrowLeft,
-  "arrow-right": arrowRight,
-  "arrow-up": arrowUp,
-  "arrow-down": arrowDown,
-  bell: bell,
-  "chevron-left": chevronLeft,
-  "chevron-right": chevronRight,
-  "chevron-up": chevronUp,
-  "chevron-down": chevronDown,
-  cross: cross,
-  copy: copy,
-  download: download,
-  upload: upload,
-  "upload-white": uploadWhite,
-  external: external,
-  globe: globe,
-  hamburger: hamburger,
-  home: home,
-  "mini-menu": miniMenu,
-  print: print,
-  save: save,
-  search: search,
-  send: send,
-  share: share,
-  star: star,
-  tick: tick,
-  warning: warning,
-  facebook: facebook,
-  twitter: twitter,
-  linkedin: linkedin,
-  instagram: instagram,
-  reload: reload,
-  equipment: equipment,
-  "equipment-white": equipmentWhite,
-  "reload-white": reloadWhite,
-  "arrow-left-white": arrowLeftWhite,
-  "arrow-right-white": arrowRightWhite,
-  "arrow-up-white": arrowUpWhite,
-  "arrow-down-white": arrowDownWhite,
-  "bell-white": bellWhite,
-  "chevron-left-white": chevronLeftWhite,
-  "chevron-right-white": chevronRightWhite,
-  "chevron-up-white": chevronUpWhite,
-  "chevron-down-white": chevronDownWhite,
-  "cross-white": crossWhite,
-  "copy-white": copyWhite,
-  "download-white": downloadWhite,
-  "external-white": externalWhite,
-  "globe-white": globeWhite,
-  "hamburger-white": hamburgerWhite,
-  "home-white": homeWhite,
-  "mini-menu-white": miniMenuWhite,
-  "print-white": printWhite,
-  "save-white": saveWhite,
-  "search-white": searchWhite,
-  "send-white": sendWhite,
-  "share-white": shareWhite,
-  "star-white": starWhite,
-  "tick-white": tickWhite,
-  "warning-white": warningWhite,
-  "facebook-white": facebookWhite,
-  "twitter-white": twitterWhite,
-  "linkedin-white": linkedinWhite,
-  "instagram-white": instagramWhite,
-  "question-mark": questionMark,
-  "question-mark-white": questionMarkWhite,
-  sidebar: sidebar,
-  "sidebar-white": sidebarWhite,
-  acorn: acorn,
-  "acorn-white": acornWhite,
-  padlock: padlock,
-  "ai-lesson": aiLesson,
+type SVGImport = {
+  src: string;
+  height: number;
+  width: number;
+  blurDataURL: string;
+  blurWidth: number;
+  blurHeight: number;
+};
```

Comment:
This approach does work, but there should be a way to set the type for svg imports. I know there is in webpack, not sure how to do it with turbopack